### PR TITLE
Task new branch merge to dev

### PR DESCRIPTION
**Pull request by necr0coder**
Merging task new branch to dev

### CODE REVIEW

**Comment by jrhurleycode** [comment]
my comment

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>test</title>
+</head>
+<body>
+    <h1>Hello World!</h1>
+    <h2>Hello again!</h2>
```
Comment (jrhurleycode) [inline index.html:10]: hello there!

**Review by jrhurleycode — COMMENTED** [review]
looks good!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>test</title>
```
Comment (jrhurleycode) [inline index.html:6]: I don't like this title.  do better.

Comment (necr0coder) [reply index.html:6]: ok I'll change it.   .....later

**Review by jrhurleycode — APPROVED** [review]
looks good

**Pull request merged**